applications-frontend/src/components/Applicant.vue

```diff
@@ -1,61 +1,123 @@
-                        </div>
-                    </div>
-                </form>
-            </aside>
-
-            <div class="column is-three-quarters">
-                <!-- random demographic information, can find a nice way to represent or table it -->
-                <section class="basicinfo">
-                    <h1 class="title">John Wick</h1>
-                    <h2 class="subtitle">Baba Yaga</h2>
-                </section>
-                <!-- responses go here -->
-                <section class="responses">
-                
-                </section>
-
-                <!-- random web links can go here -->
-                <section class="links">
-
-                </section>
-            </div>
-        </div>
-    </div>
+<div class="container">
+	<div class="columns is-multiline">
+
+		<!-- Control panel -->
+		<div class="column section-column has-background-white is-4">		
+			<center>
+	
+				<!-- Rating control -->
+				<div class="subtitle is-3">Rating</div>
+
+				<div class="columns is-mobile is-multiline has-background-light is-gapless">
+					<div class="column">						
+						<label class="control control--radio column  is-flex is-align-center is-flex-column-reverse" >1
+						<input v-on:click="current_rating = 1" type="radio" name="radio" checked="checked"/>
+						<div class="control__indicator"></div>
+						</label>							
+					</div>	
+
+					<div class="column">
+						<label class="control control--radio column  is-flex is-align-center is-flex-column-reverse" >2
+						<input v-on:click="current_rating = 2" type="radio" name="radio" checked="checked"/>
+						<div class="control__indicator"></div>
+						</label>	
+					</div>	
+
+					<div class="column">
+						<label class="control control--radio column  is-flex is-align-center is-flex-column-reverse" >3
+						<input v-on:click="current_rating = 3" type="radio" name="radio" checked="checked"/>
+						<div class="control__indicator"></div>
+						</label>	
+					</div>	
+
+					<div class="column">
+						<label class="control control--radio column  is-flex is-align-center is-flex-column-reverse" >4
+						<input v-on:click="current_rating = 4" type="radio" name="radio" checked="checked"/>
+						<div class="control__indicator"></div>
+						</label>	
+					</div>	
+
+					<div class="column">
+						<label class="control control--radio column  is-flex is-align-center is-flex-column-reverse" >5
+						<input  v-on:click="current_rating = 5" type="radio" name="radio" checked="checked"/>
+						<div class="control__indicator"></div>
+						</label>	
+					</div>					
+				</div>
+				
+				<!-- Next applicant button -->
+				<a v-on:click="next_applicant" class="button is-primary is-medium" style="margin-top: 1em;">Next applicant</a>				
+			</center>		
+		</div>
+
+		<!-- Information panel -->
+		<div class="column has-background-light" style="border-radius: 1em;">
+
+			<div class="columns is-centered is-multiline">
```

Comment:
two things: can we add a h2 that says "information", and these shouldn't all be their own column, can you make just two columns and then just add the data to each? That would be better